docs/Tutorials/Create-Private-Clique-Network.md

```diff
@@ -0,0 +1,271 @@
+
+!!!important
+    An Ethereum private network created as described here is isolated but not protected or secure. 
+    We recommend running the private network behind a properly configured firewall.
+
+## Prerequisites 
+
+[Pantheon](../Installation/Install-Binaries.md) 
+
+[Curl (or similar web service client)](https://curl.haxx.se/download.html) 
+
+## Steps
+
+To create a private network: 
+
+1. [Create Folders](#1-create-folders)
+1. [Get Public Key for Node-1](#2-get-public-key-for-node-1)
+1. [Get Address for Node-1](#3-get-address-for-node-1)
+1. [Create Genesis File](#4-create-genesis-file)
+1. [Delete Database Directory](#5-delete-database-directory)
+1. [Start First Node as Bootnode](#6-start-first-node-as-bootnode)
+1. [Start Node-2](#7-start-node-2)
+1. [Start Node-3](#8-start-node-3)
+1. [Confirm Private Network Working](#9-confirm-private-network-is-working)
+
+### 1. Create Folders 
+
+Each node requires a data directory for the blockchain data. When the node is started, the node key is saved in this directory. 
+
+Create directories for your private network, each of the three nodes, and a data directory for each node: 
+
+```bash
+Clique-Network/
+├── Node-1
+│   ├── Node-1-data-path
+├── Node-2
+│   ├── Node-2-data-path
+└── Node-3
+    ├── Node-3-data-path
+```
+
+### 2. Get Public Key for Node-1
+
+To enable nodes to discover each other, a network requires one or more nodes to be bootnodes. 
+For this private network, we will use Node-1 as the bootnode. This requires obtaining the public key for the enode URL. 
+
+In the `Node-1` directory, use the [`export-pub-key`](../Reference/Pantheon-CLI-Syntax.md#export-pub-key) subcommand to write 
+the [node public key](../Configuring-Pantheon/Node-Keys.md) to the specified file (`publicKeyNode1` in this example):
+
+```bash tab="MacOS"
+pantheon --data-path=Node-1-data-path public-key export --to=Node-1-data-path/publicKeyNode1
+```
+
+```bash tab="Windows"
+pantheon --data-path=Node-1-data-path public-key export --to=Node-1-data-path\publicKeyNode1
+```
+
+Your node 1 directory now contains: 
+```bash
+├── Node-1
+    ├── Node-1-data-path
+        ├── database
+        ├── key
+        ├── publicKeyNode1
+```
+      
+The `database` directory contains the blockchain data. 
+
+### 3. Get Address for Node-1 
+
+In Clique networks, the address of at least one initial signer must be included in the genesis file. 
+For this Clique network, we will use Node-1 as the initial signer. This requires obtaining the address for Node-1. 
+
+To obtain the address for Node-1, do one of the following with the private key in the `key` file in the `Node-1-data-path` directory: 
+
+* Import the private key into [MetaMask](https://metamask.io/) and click the **Copy to clipboard** button 
+displayed when hovering over the account name and address. 
+* Use the [ethereumjs-util](https://github.com/ethereumjs/ethereumjs-util) library and [node](https://nodejs.org/en/) 
```

Comment:
shouldn't we restric the manipulation on the private key and favour the use of the public one to generate addresses ? See https://github.com/ethereumjs/ethereumjs-util/blob/master/docs/README.md#pubtoaddress

Reply:
I used MetaMask, I didn't test the ethereumjs-util bit

Reply:
@NicolasMassart - what's the rationale for that approach? You have to import the private key to MetaMask so we'd have private key for one scenario and public key for the other.

Reply:
This will be updated when new public key to address command created. Leaving as is for now.